### PR TITLE
New aport libp8-platform

### PR DESCRIPTION
**Pull request by vrnagy**
Platform support library used by libCEC and binary add-ons for Kodi
Required for newer libcec library

### CODE REVIEW

**Comment by kaniini** [comment]
Please update the commit to be of the format: `community/libp8-platform: new aport (dependency of libcec)`, thanks

File: testing/libp8-platform/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+arch="all"
+license="GPL"
+depends=""
+depends_dev=""
+makedepends="$depends_dev automake libtool cmake"
+install=""
+subpackages="$pkgname-dev"
+source="https://github.com/Pulse-Eight/platform/archive/p8-platform-$pkgver.tar.gz"
+
+_builddir="$srcdir"/platform-p8-platform-$pkgver
```
Comment (kaniini) [inline testing/libp8-platform/APKBUILD]: should be `$builddir` in modern APKBUILDs.

File: testing/libp8-platform/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+build() {
+	mkdir "$_builddir/build"
+	cd "$_builddir/build"
+	cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=1 -DCMAKE_INSTALL_PREFIX=/usr ..
```
Comment (kaniini) [inline testing/libp8-platform/APKBUILD]: should `|| return 1` on the cmake.

File: testing/libp8-platform/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+pkgname=p8-platform
+pkgver=2.1.0.1
+pkgrel=1
+pkgdesc="Platform support library used by libCEC and binary add-ons for Kodi"
+url="https://github.com/Pulse-Eight/platform"
+arch="all"
+license="GPL"
+depends=""
+depends_dev=""
+makedepends="$depends_dev automake libtool cmake"
```
Comment (kaniini) [inline testing/libp8-platform/APKBUILD]: why are automake and libtool dependencies since the project uses cmake?

**Comment by vrnagy** [comment]
Fixed as requested

**Comment by vrnagy** [comment]
@kaniini Can you please take a look at it?

**Pull request closed without merging**